webapp/static/interop-data.json

```diff
@@ -322,119 +322,118 @@
                 "countsTowardScore": true
             },
             "interop-2023-fonts": {
                 "description": "Font Feature Detection and Palettes",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-fonts", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/font-palette",
+                "spec": "https://w3c.github.io/csswg-drafts/css-fonts/#font-palette-prop",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-fonts",
                 "countsTowardScore": true
             },
             "interop-2023-forms": {
                 "description": "Forms",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-forms", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form",
+                "spec": "https://html.spec.whatwg.org/multipage/forms.html#the-form-element",
                 "countsTowardScore": true
             },
             "interop-2023-grid": {
                 "description": "Grid",
-                "mdn": "",
-                "spec": "",
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout",
+                "spec": "https://drafts.csswg.org/css-grid/",
                 "tests": "/results/css/css-grid?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2021-grid%20or%20label%3Ainterop-2023-grid", 
                 "countsTowardScore": true
             },
             "interop-2023-has": {
                 "description": ":has()",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-has", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/:has",
+                "spec": "https://drafts.csswg.org/selectors-4/#relational",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-has",
                 "countsTowardScore": true
             },
             "interop-2023-inert": {
                 "description": "Inert",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-inert", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert",
```

Comment:
The global attribute page might be better: https://developer.mozilla.org/docs/Web/HTML/Global_attributes/inert